yarn.lock

```diff
@@ -1212,12 +1212,7 @@
     find-up "^4.1.0"
     fs-extra "^8.1.0"
 
-"@openzeppelin/contracts@^3.3.0":
```

Comment:
@GalloDaSballo wanted to call out this dependency change

Reply:
These have known vulnerabilities:
https://security.snyk.io/vuln?search=OpenZeppelin%2Fcontracts

Reply:
We should use: https://www.npmjs.com/package/@openzeppelin/contracts/v/4.9.5

Reply:
Well known, bug fixed, still supported

Reply:
thanks, fixed version @ 4.9.5